### PR TITLE
HDS Website -- Addition of UI Libraries

### PR DESCRIPTION
**Pull request by majedelass**
### :pushpin: Summary

If merged, this PR will add the links of our A11Y, Utilities and Wireframe libraries to our website in the "getting started for designers" section. Brief descriptions were added for each link.

### :camera_flash: Screenshots

box with a red border emphasizing the new links added to the section

***

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

File: website/docs/getting-started/for-designers.md [hunk 1]
```diff
@@ -181,6 +181,9 @@ HashiCorp product teams can use the Helios libraries published under the [HDS De
 - [Product Components](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?t=Ooe3pkDap3cGcgAH-1): the set of components published in the Helios Design System
 - [Product Foundations](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?t=4kdgl88SMIiEYhbA-1): core styles including typography, color, and elevation that are consumed by Helios components for reuse in your projects
 - [Icons](https://www.figma.com/file/TLnoT5AYQfy3tZ0H68BgOr/Flight-Icons?t=nEh4FAxdjRsVInyL-1): core icon set for reuse in your projects
+- [A11Y Helper Annotation Kit](https://www.figma.com/design/EEfM1jjqLfUv59eYLopn5x/HDS-A11Y-Helper-UI-Kit?m=auto&t=JLqmUYB9fZbdWaa2-6): an annotation toolkit designed to enhance collaboration between designers and egineers, ensuring the work meets accessibility (a11y) standards.
```
Comment (LilithJames-HDS) [inline website/docs/getting-started/for-designers.md:184]:
```suggestion
- [A11Y Helper Annotation Kit](https://www.figma.com/design/EEfM1jjqLfUv59eYLopn5x/HDS-A11Y-Helper-UI-Kit?m=auto&t=JLqmUYB9fZbdWaa2-6): an annotation toolkit designed to enhance collaboration between designers and engineers, ensuring the work meets accessibility (a11y) standards.
```

File: website/docs/getting-started/for-designers.md [hunk 2]
```diff
@@ -181,6 +181,9 @@ HashiCorp product teams can use the Helios libraries published under the [HDS De
...
+- [Wireframe UI Kit](https://www.figma.com/design/w0ukydeAsbv6sJirLxZMBo/HDS-Wireframes?m=auto&t=JLqmUYB9fZbdWaa2-6): This library provides low-fidelity versions of HDS components designed to help quickly sketch ideas and build out flows.
```
Comment (heatherlarsen) [inline website/docs/getting-started/for-designers.md:185]:
```suggestion
- [Wireframe UI Kit](https://www.figma.com/design/w0ukydeAsbv6sJirLxZMBo/HDS-Wireframes?m=auto&t=JLqmUYB9fZbdWaa2-6): low-fidelity versions of HDS components designed to help quickly sketch ideas and build flows.
```

Comment (LilithJames-HDS) [inline website/docs/getting-started/for-designers.md:185]:
```suggestion
- [Wireframe UI Kit](https://www.figma.com/design/w0ukydeAsbv6sJirLxZMBo/HDS-Wireframes?m=auto&t=JLqmUYB9fZbdWaa2-6): low-fidelity versions of HDS components designed to help quickly sketch ideas and build out flows.
```

File: website/docs/getting-started/for-designers.md [hunk 3]
```diff
@@ -181,6 +181,9 @@ HashiCorp product teams can use the Helios libraries published under the [HDS De
...
+- [Utility UI Kit](https://www.figma.com/design/vqhh0wWZ8PwnRkvh1jk7jB/Utilities-UI-Kit?m=auto&t=JLqmUYB9fZbdWaa2-6): This library includes documentation styles, helpers, annotations, component acceptance criteria, and external utilities like browser scroll bars.
```
Comment (LilithJames-HDS) [inline website/docs/getting-started/for-designers.md:186]:
```suggestion
- [Utility UI Kit](https://www.figma.com/design/vqhh0wWZ8PwnRkvh1jk7jB/Utilities-UI-Kit?m=auto&t=JLqmUYB9fZbdWaa2-6): documentation styles, helpers, annotations, component acceptance criteria, and external utilities like browser scroll bars.
```

**Review by LilithJames-HDS — CHANGES_REQUESTED** [review]
One spelling catch then just making parallel structure for the last two bullets. Thanks for adding these to the site!

File: website/docs/getting-started/for-designers.md (hunk 3 above)
Comment (heatherlarsen) [inline website/docs/getting-started/for-designers.md:186]:
```suggestion
- [Utility UI Kit](https://www.figma.com/design/vqhh0wWZ8PwnRkvh1jk7jB/Utilities-UI-Kit?m=auto&t=JLqmUYB9fZbdWaa2-6): common UI helpers like documentation styles and annotations, and external utilities like browser scroll bars and cursors.
```

**Review by heatherlarsen — APPROVED** [review]
Looks great, just left to very nit pick comments.

File: website/docs/getting-started/for-designers.md (hunk 2 above)
Comment (KristinLBradley) [reply website/docs/getting-started/for-designers.md:185]: I would change:
"sketch ideas and build out flows"

to:
"sketch out ideas and build flows"

**Review by KristinLBradley — APPROVED** [review]
Added one minor suggestion. Otherwise looks good and I agree with others suggested changes.

**Review by LilithJames-HDS — APPROVED** [review]

**Pull request merged**